fmriprep/workflows/bold/base.py

```diff
@@ -199,16 +199,16 @@ def init_func_preproc_wf(
 
         bold_file
             BOLD series NIfTI file
-        t1_preproc
+        t1w_preproc
             Bias-corrected structural template image
-        t1_brain
-            Skull-stripped ``t1_preproc``
-        t1_mask
+        t1w_brain
+            Skull-stripped ``t1w_preproc``
+        t1w_mask
             Mask of the skull-stripped template image
-        t1_seg
+        t1w_dseg
```

Comment:
`t1w_aseg` and `t1w_aparc` are missing here

Reply:
sorry, I missed this one. I'll try to get at this asap

Reply:
done (7e5b2f1288f53ed54413d997a615a91a999a3597)